components/dropdownlist/data-bind.md

```diff
@@ -51,14 +51,15 @@ To bind the DropDownList to a model:
 
 1. populate its `Data` property with the collection of items you want in the dropdown
 1. set the `TextField` and `ValueField` properties to point to the corresponding names of the model
-1. set the `Value` property to the intial value of the model. If not set, it will be populated with the first item in the data source.
+2. set the `Value` property to the initial value of the model. If not set, it will be populated with the first item in the data source or the `DefaultText`.
```

Comment:
The Value will not be populated with the DefaultText, that's just an appearance feature. The Value can be null or the default value of the type. Please also check the Considerations section and let me know if you still find the docs lacking https://docs.telerik.com/blazor-ui/components/dropdownlist/data-bind#considerations